tests/test_xontribs.py

```diff
@@ -1,6 +1,56 @@
 """xontrib tests, such as they are"""
-from xonsh.xontribs import xontrib_metadata
+import pytest
+from xonsh.xontribs import xontrib_metadata, xontrib_context
+import sys
```

Comment:
This doesn't block because it is the tests, but PEP8 says that imports should be grouped.